internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -609,6 +609,18 @@ func convertEntryPoint(entrypoint manifest.EntryPointOverride) ([]string, error)
 	return out, nil
 }
 
+func getDeployConfig(deployStrategyName string) template.DeploymentConfigurationOpts {
+	var deployConfigs template.DeploymentConfigurationOpts
+	if deployStrategyName == "recreate" {
```

Comment:
nit: we can have a constant (maybe we already do from the last PR?) in `manifest` package `const deploymentStrategyRecreate = "recreate". And here we'd have:
```suggestion
	if deployStrategyName == manifest.deploymentStrategyRecreate {
```

Reply:
So we have `manifest.ecsRecreateRollingUpdateStrategy = "recreate"` from the last PR but as it is not exported we can not use it in the `transformers.go`. 
I can change it to `EcsRecreateRollingUpdateStrategy` in the manifest and use it in the `transformers.go` as `manifest.EcsRecreateRollingUpdateStrategy`

Reply:
Let's do it! Maybe instead of `Ecs` we can write `ECS`!